content/en/users/cloud-compute/openstack/_index.md

```diff
@@ -29,14 +29,39 @@ of this guide will focus on CLI/API access.
 
 ## Installation
 
-TBC
+The OpenStack client is a command-line client for OpenStack that brings the
+command set for Compute, Identity, Image, Object Storage and Block Storage
+APIs together in a single shell with a uniform command structure.
+
+> If installing on Windows, as there are non-pure Python packages needed for
+> installation, please make sure [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/)
+> are installed, with the following options selected:
+>
+> - C++ CMake tools for Windows
+> - C++ ATL for latest v142 build tools (x86 & x64)
+> - Testing tools core features - Build Tools
+> - Windows 10 SDK (`<latest`>)
+>
+> In case you prefer to use non-Microsoft alternatives for building non-pure
+> packages, please see [here](https://wiki.python.org/moin/WindowsCompilers).
+
+Installation of the OpenStack client can be done using:
 
 ```shell
+pip install --upgrade setuptools
 pip install requests
```

Comment:
Probably this also should be removed, it comes as a dependency

Reply:
apparently, a lot of people had problems installing the openstackclient package, because of two things: they did not have the  ms build tools, and then they had an older python installation that was not able to properly build the non-pure packages required for openstack. this ensures that the setuptools are up to date. sure, if you start from a fresh Python installation, you don't need this.

Reply:
I would also remove the explicit request install, but if you feel you could mention something about problems/solutions in the quote for windows. 
Or maybe adding a subsection covering some specificities when installing on windows (what would be nice would be some tabs allowing to have // doc for installing on Windows, Linux or Mac that the user could select, but I don't think we currently have an easy/simple way to do this).

I think all this probably haven't been tested much on Windows, so it's certainly helpful to have this be done.